oshi-core/src/main/resources/oshi.properties

```diff
@@ -116,12 +116,17 @@ oshi.os.windows.perfproc.disabled=
 # PerfDisk counters used for HWDiskStore reads/writes/queue length/xfer time
 oshi.os.windows.perfdisk.disabled=
 
-# On Linux, most process metrics are read from the proc pseudo-filesystem. 
+# On Linux, most process metrics are read from the proc pseudo-filesystem.
 # When operating without elevated permissions, this results in frequent error
 # messages for failures to read the process environment files. Set this to true
 # to receive these warnings.
 oshi.os.linux.procfs.logwarning=false
 
+#Macos gives log warning if SYSCTL calls fails
+#In order to avoid warning make this property true
+
```

Comment:
Since the reasoning for this is the same as Linux (no elevated permissions) can we merge the two paragraphs above?  For example (feel free to use different wording) insert a comment right below line 119, `# On macOS, process environment is read via sysctl`. And then you can put the two settings next to each other.

Reply:
Ya thanks for the suggesstion.